finance/accounting.md

```diff
@@ -1,52 +1,68 @@
 # Accounting data
 
 Accounting data describes the money that we **know** we have spent or that we **highly expect** to receive.
-There are a few kinds of accounting data we have access to:
 
-All 2i2c-specific accounting documents should be accessible on our Team Drive to anybody with a `@2i2c.org` address.
-We don't control the access conditions for the CS&S documents described below, so if you don't have access to these, ask the {role}`Executive Director`.
+## Accounting transactions airtable
 
-We split our financial and accounting data between two places:
+{term}`CS&S` provides us a monthly dataset of the last 6 months of all of all our accounting transactions.
+This is [in a Google Sheet that is automatically updated each day][gsheet].
+It includes the last 6 months of any transaction 2i2c has made (if we need more than 6 months of history, ask `fsp@2i2c.org` for a custom data dump).
 
-- [2i2c Drive Folder with our finance assets](https://drive.google.com/drive/folders/1D5NQKhPDP6zMQ8EdLcMOceTz-ek81nmX?usp=sharing) contains 2i2c-specific budgeting and modeling information. It is available to all team members.
+We have a copy of this sheet in an [**Accounting Transactions AirTable**][accounting-table][^airtable].
+Any columns that are synchronized with CS&S have a little lightning bolt (⚡) next to them.
```

Comment:
I did not see any lightning bolts ⚡ in columns in the Accounting Transactions AirTable. Perhaps the lightning bolts ⚡ appear in the Google sheet and not in the copy hosted in AirTable?

Reply:
ah that's a good point, the lightning bolts are actually only in the `Contracts` and `Invoices` sections, because this data is all manually updated. I'll move this note over there.